Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs

```diff
@@ -29,7 +29,16 @@ internal class SimulatedHandState
         // Activate the pinch gesture
         // Pinch is a special gesture that triggers the Select and TriggerPress input actions
         // The pinch action doesn't occur until the gesture is completed.
-        public bool IsPinching => gesture == ArticulatedHandPose.GestureId.Pinch && gestureBlending == 1.0f;
+        public bool IsPinching
+        {
+            get
+            {
+                if (handedness == Handedness.None)
```

Comment:
Should `gestureBlending` for gaze just always be 1.0? There's not much for it to transition between, since there's no physical representation like there is for hands.

Reply:
it seems wrong to override a bunch of code around setting and getting the `gestureBlending` variable to ensure that it stays fixed at 1.0f when we're using the hands free input simulation. I think its better to let `gestureBlending` be set and managed as usual and just target how isPinching specifically behaves.

Reply:
Yeah, that seems good!

My thought was around removing the line where we cycle through the gaze blending:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/99b0f02b76fd7b5b13e8d9c8c35729af6e307f2e/Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs#L335

and set it to 1.0f, like `Reset` does, and leave it:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/99b0f02b76fd7b5b13e8d9c8c35729af6e307f2e/Assets/MRTK/Core/Providers/InputSimulation/SimulatedHandDataProvider.cs#L109

Reply:
Oh! Missed the parts where setting the gesture property resets it to 0.0f. Yeah, bigger change than I thought.